x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_columns.tsx

```diff
@@ -0,0 +1,357 @@
+        field: 'fieldValue',
+        name: i18n.translate('xpack.aiops.logRateAnalysis.resultsTable.fieldValueLabel', {
+          defaultMessage: 'Field value',
+        }),
+        render: (_, { fieldValue, type }) => (
+          <span title={String(fieldValue)}>
+            {type === 'keyword' ? (
+              String(fieldValue)
+            ) : (
+              <EuiText size="xs">
+                <EuiCode language="log" transparentBackground css={{ paddingInline: '0px' }}>
+                  {String(fieldValue)}
+                </EuiCode>
+              </EuiText>
+            )}
+          </span>
+        ),
+        sortable: true,
+        textOnly: true,
+        truncateText: { lines: TRUNCATE_TEXT_LINES },
+        valign: 'middle',
+      },
+      ['Log rate']: {
+        'data-test-subj': 'aiopsLogRateAnalysisResultsTableColumnLogRate',
+        width: NARROW_COLUMN_WIDTH,
+        field: 'pValue',
+        name: (
+          <>
+            <FormattedMessage
+              id="xpack.aiops.logRateAnalysis.resultsTable.logRateLabel"
+              defaultMessage="Log rate"
+            />
+            &nbsp;
+            <EuiIconTip
+              size="s"
+              position="top"
+              color="subdued"
+              type="questionInCircle"
+              className="eui-alignTop"
+              content={isGroupsTable ? groupLogRateHelpMessage : logRateHelpMessage}
+            />
+          </>
+        ),
+        render: (_, { histogram, fieldName, fieldValue }) => (
+          <MiniHistogram
+            chartData={histogram}
+            isLoading={loading && histogram === undefined}
+            label={`${fieldName}:${fieldValue}`}
+            barColorOverride={barColorOverride}
+            barHighlightColorOverride={barHighlightColorOverride}
+          />
+        ),
+        sortable: false,
+        valign: 'middle',
+      },
+      ['Impact']: {
+        'data-test-subj': 'aiopsLogRateAnalysisResultsTableColumnImpact',
+        width: NARROW_COLUMN_WIDTH,
+        field: 'pValue',
+        name: (
+          <>
+            <FormattedMessage
+              id="xpack.aiops.logRateAnalysis.resultsTable.impactLabel"
+              defaultMessage="Impact"
+            />
+            &nbsp;
+            <EuiIconTip
+              size="s"
+              position="top"
+              color="subdued"
+              type="questionInCircle"
+              className="eui-alignTop"
+              content={isGroupsTable ? groupImpactMessage : impactMessage}
+            />
+          </>
+        ),
+        render: (_, { pValue }) => {
+          if (typeof pValue !== 'number') return NOT_AVAILABLE;
+          const label = getFailedTransactionsCorrelationImpactLabel(pValue);
+          return label ? <EuiBadge color={label.color}>{label.impact}</EuiBadge> : null;
+        },
+        sortable: true,
+        valign: 'middle',
+      },
+      ['p-value']: {
```

Comment:
The `p-value` column should be hidden by default.

Reply:
Good catch - updated in 72f120e9a39c579999d4b3aabcf548ff88d6fdf0